### PR TITLE
fix(curriculum/ python): flesh out block pages

### LINKED ISSUE #43289: Block pages should be fleshed out
## Is your feature request related to a problem? Please describe.

The block level pages are both orphaned (there are no links to them) and don't fit with the style of the superblock pages.  e.g. https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/ vs https://www.freecodecamp.org/learn/responsive-web-design/

## Describe the solution you'd like

These pages should match the design of the rest of the site and be translatable.

## Describe alternatives you've considered

Removing the pages entirely.

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes part of #43289 

Python practice projects list to be fleshed out:

- [x] Learn Python List Comprehensions by Building a Case Converter Program.
- [x] Learn String Manipulation by Building a Cipher	
- [x] Learn How to Work with Numbers and Strings by Implementing the Luhn Algorithm	
- [x] Learn Lambda Functions by Building a Expense Tracker	
- [x] Learn numerical analysis by finding the square root of a number using the bisection method		
- [x] Learn Regular Expressions by Building a Password Generator	
- [x] Learn Algorithm Design by Building the Shortest Path Algorithm	
- [x] Learn Recursion by Solving the Tower of Hanoi Puzzle	
- [x] Learn Data Structures by Building the Merge Sort Algorithm 		
- [x] Learn Tree Traversal by Building a Binary Search Tree	
- [x] Learn Special Methods by Building a Vector Space	
- [ ] Learn Interfaces by Building an Equation Solver
- [ ] Learn Encapsulation by Building a Projectile Trajectory Calculator

Certification Projects:
- [ ] Build a Probability Calculator Project
- [ ] Build a Polygon Area Calculator Project
- [ ] Build a Budget App Project
- [ ] Build a Time Calculator Project
- [ ] Build an Arithmetic Formatter Project

### CODE REVIEW

**Review by moT01 — COMMENTED** [review]
I'm not sure exactly what we want here @zairahira, but I think we should try to come up with something a little longer. It would be nice to have a paragraph or two for each project.

Perhaps this would be good enough for now, as it is an improvement.

**Comment by zairahira** [comment]
I have used the same description as the existing projects to get rid of the default line. 
We can definitely work on adding an extended description.

**Comment by Dario-DC** [comment]
What about if we use the intros from the `intro.json` @moT01 ? For example:
```
"intro": [
          "Python is a powerful and popular programming language widely used for data science, data visualization, web development, game development, machine learning and more.",
          "In this project, you'll learn fundamental programming concepts in Python, such as variables, functions, loops, and conditional statements. You'll use these to code your first programs."
        ]
```

**Review by fhsinchy — APPROVED** [review]

**Comment by zairahira** [comment]
@moT01 and @naomi-lgbt, I have attempted to update the `index.md` for the shortest path algorithm. Please share your views on it.

File: client/src/pages/learn/scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/index.md
Comment (naomi-lgbt) [inline client/src/pages/learn/scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/index.md:1]: Yes! This is an excellent starting point!

**Review by moT01 — COMMENTED** [review]
This is looking good @zairahira 🎉 

Is it ready to go or do you want to fill in the few missing ones? I would be fine getting it in as-is since it's quite an improvement and we can work on the others when we get a chance.

**Comment by zairahira** [comment]
We can work on the remaining ones later on. Let's merge this one then.

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**